src/main/kotlin/no/skatteetaten/aurora/boober/utils/jsonNodeUtils.kt

```diff
@@ -164,9 +164,13 @@ fun JsonNode.mergeField(source: ObjectNode, root: String, field: String) {
     rootNode.replace(field, mergedObject)
 }
 
-fun JsonNode?.startsWith(pattern: String, message: String): Exception? {
+fun JsonNode?.startsWith(pattern: String, message: String, required: Boolean = true): Exception? {
     if (this == null || !this.isTextual) {
-        return IllegalArgumentException(message)
+        return if (required) {
```

Comment:
Ender man ikke her opp med å tillate ugyldige ikke-tekstlige verdier for config som ikke er påkrevet?

Reply:
Jo, men jeg har bare gjort som alle andre steder. Jeg vet ikke hvorfor det er slik. Men akkurat her kan jeg trygt sjekke på isTextual først, ja.